packages/runner/src/scanner/scanner.ts

```diff
@@ -16,11 +16,8 @@ export class Scanner {
 
             await this.page.create();
             await this.page.enableBypassCSP();
-            await this.page.goto(url);
 
-            const result = await this.page.scanForA11yIssues();
-
-            return { results: result };
+            return this.page.scanForA11yIssues(url);
```

Comment:
Use return await otherwise catch will never gets executed. Here is more info 
https://jakearchibald.com/2017/await-vs-return-vs-return-await/

Reply:
Thanks, good catch!